Konstantin_Sudakov/loger.js

```diff
@@ -0,0 +1,84 @@
+
+//Модули для работы
+const fs = require('fs');
+const { promisify } = require('util');
+const promisifiedReadFile = promisify(fs.readFile);
+const minimist = require('minimist');
+const argv = minimist(process.argv.splice(2));
+
+//переменные для работы
+let fileForWrite = argv._[0];
+let wins = 0;
+let losing = 0;
+let games = 0;
+let winsMax = 0;
+let losingMax = 0;
+
+//мои функции для вычислений
+const MakeStatistics = (data) => {
+    const arr = data.split("");
+    console.log(arr);
+    for (let i = 0; i<arr.length; i++) {
+        if (arr[i] == 0) {
+            losing++;
+        }
+        if (arr[i] == 1) {
+           wins++;
+        }
+        games++;
+    }
+    getMax(data, '1');
+    getMax(data, '0');
+    getStatistics();
+
+};
+const getMax = (arr, index) => {
+    const arr2 = arr.split(index);
+    for(let i = 0 ; i < arr2.length; i++) {
+        let maxArr = arr2[i].split("");
+        console.log(maxArr);
+        if (index === '1') {
+            if (maxArr.length > losingMax) {
+                losingMax = maxArr.length
+            }
+        } else if (index = '0') {
+            if (maxArr.length > winsMax) {
+                winsMax = maxArr.length
+            }
+        }
+
+    }
+};
+const getStatistics = () => {
+    const a = wins/games*100;
```

Comment:
Не забывайте поддерживать чистоту и читаемость кода
```suggestion
    const a = wins / games * 100;
```